NCATS_Data_Dictionary.md

```diff
@@ -282,28 +282,36 @@ notification refers to
 ### places Collection ###
 
 The data in this collection is derived from the "Government Units" and
-"Populated Places" Topical Gazetteers files from
-[USGS](https://geonames.usgs.gov/domestic/download_data.htm).
+"Populated Places" [Topical Gazetteers files](https://prd-tnm.s3.amazonaws.com/index.html?prefix=StagedProducts/GeographicNames/Archive/)
+from [USGS](https://www.usgs.gov/us-board-on-geographic-names/download-gnis-data).
 
-- `_id` [long integer]: [GNIS
-  ID](https://geonames.usgs.gov/domestic/index.html) corresponding to
-  this place
+- `_id` [long integer]: [Feature
+  ID](https://www.usgs.gov/us-board-on-geographic-names/domestic-names) corresponding
+  to this place
+- `name` [string]: Full name of this place
 - `class` [string]: Class of this place ("COUNTY", "STATE", "Populated
   Place", "Civil")
-- `country` [string]: Two-letter abbreviation of the country where this
-  place is
-- `country_name` [string]: Full name of the country where this place is
-- `county` [string]: Full name of the county where this place is
-- `county_fips` [string]: [FIPS
-  code](https://catalog.data.gov/dataset/fips-county-code-look-up-tool)
-  for the county where this place is
-- `name` [string]: Full name of this place
 - `state` [string]: Two-letter postal abbreviation of the state where
   this place is
 - `state_fips` [string]: [FIPS
-  code](https://catalog.data.gov/dataset/fips-state-codes) for the
-  state where this place is
+  code](https://www.census.gov/library/reference/code-lists/ansi.html#states)
+  for the state where this place is
 - `state_name` [string]: Full name of the state where this place is
+- `county` [string]: Full name of the county where this place is
+- `county_fips` [string]: [FIPS
+  code](https://www.census.gov/library/reference/code-lists/ansi.html#cou)
+  for the county where this place is
+- `country` [string]: Two-letter abbreviation of the country where this
+  place is
+- `country_name` [string]: Full name of the country where this place is
+- `latitude_dms` [string]: Latitude of this place, in degrees, minutes,
+  and seconds
+- `longitude_dms` [string]: Longitude of this place, in degrees,
+  minutes, and seconds
```

Comment:
Should we specify the format of the DMS string?  I've seen it done a few different ways, e.g., `DDDMMSS`, `DDD MM SS`.

Reply:
Please let me know if https://github.com/cisagov/ncats-data-dictionary/pull/43/commits/c0b7a4c67e6ecd8fd4528a83e3e8d944ecaae365 works for you. I also added https://github.com/cisagov/ncats-data-dictionary/pull/43/commits/857a112a278a0a90d5ebe5dfc0eff6caabad92f8 as a more general catch-all for the formats of this collection's fields.

Reply:
Perfect!